### ISSUE TITLE
bubble_convergence fails with PPM

### ISSUE THREAD

**Issue by zingale** [AMReX-Astro/Castro#826]
If you build the `bubble_convergence` problem as:
```
make DEBUG=TRUE USE_REACT=FALSE -j 20
```

and run as:
```
./Castro2d.gnu.DEBUG.MPI.ex inputs_2d.64 castro.ppm_type=1 castro.time_integration_method=0 castro.do_react=0 amr.probin_file=probin.64.nopert amrex.fpe_trap_invalid=1 castro.sdc_order
=2
```

then it dies immediately with and erroneous arithmetic error:
```
 0: ./Castro2d.gnu.DEBUG.MPI.ex(+0x2b4450) [0x56032cf69450]
    amrex::BLBackTrace::print_backtrace_info(_IO_FILE*)
/home/zingale/development/AMReX//Src/Base/AMReX_BLBackTrace.cpp:163:29

 1: ./Castro2d.gnu.DEBUG.MPI.ex(+0x2b409a) [0x56032cf6909a]
    amrex::BLBackTrace::handler(int)
/home/zingale/development/AMReX//Src/Base/AMReX_BLBackTrace.cpp:71:8

 2: /lib64/libc.so.6(+0x3c6b0) [0x7f76dd7006b0]
    __restore_rt
??:0

 3: ./Castro2d.gnu.DEBUG.MPI.ex(+0xca957) [0x56032cd7f957]
    Castro::riemannus(amrex::Box const&, amrex::Array4<double>, amrex::Array4<double>, amrex::Array4<double const>, amrex::Array4<double>, int, int)
../../../Source/hydro/riemann_solvers.cpp:637:3

 4: ./Castro2d.gnu.DEBUG.MPI.ex(+0xbdbd3) [0x56032cd72bd3]
    Castro::riemann_state(amrex::Box const&, amrex::Array4<double>, amrex::Array4<double>, amrex::Array4<double>, amrex::Array4<double const>, int, int)
../../../Source/hydro/riemann.cpp:244:14

 5: ./Castro2d.gnu.DEBUG.MPI.ex(+0xbae07) [0x56032cd6fe07]
    Castro::cmpflx_plus_godunov(amrex::Box const&, amrex::Array4<double>, amrex::Array4<double>, amrex::Array4<double>, amrex::Array4<double>, amrex::Array4<double>, amrex::Array4<doub
le const>, amrex::Array4<double const>, int)
../../../Source/hydro/riemann.cpp:39:18

 6: ./Castro2d.gnu.DEBUG.MPI.ex(+0xaf26d) [0x56032cd6426d]
    Castro::construct_ctu_hydro_source(double, double)
../../../Source/hydro/Castro_ctu_hydro.cpp:535:26

 7: ./Castro2d.gnu.DEBUG.MPI.ex(+0x69440) [0x56032cd1e440]
    Castro::do_advance_ctu(double, double, int, int)
../../../Source/driver/Castro_advance_ctu.cpp:155:36

 8: ./Castro2d.gnu.DEBUG.MPI.ex(+0x6a81c) [0x56032cd1f81c]
    Castro::subcycle_advance_ctu(double, double, int, int)
../../../Source/driver/Castro_advance_ctu.cpp:543:45

 9: ./Castro2d.gnu.DEBUG.MPI.ex(+0x66e40) [0x56032cd1be40]
    Castro::advance(double, double, int, int)
../../../Source/driver/Castro_advance.cpp:55:55

10: ./Castro2d.gnu.DEBUG.MPI.ex(+0x33945e) [0x56032cfee45e]
    amrex::Amr::timeStep(int, double, int, int, double)
/home/zingale/development/AMReX//Src/Amr/AMReX_Amr.cpp:2112:44

11: ./Castro2d.gnu.DEBUG.MPI.ex(+0x339cb6) [0x56032cfeecb6]
    amrex::Amr::coarseTimeStep(double)
/home/zingale/development/AMReX//Src/Amr/AMReX_Amr.cpp:2439:26

12: ./Castro2d.gnu.DEBUG.MPI.ex(+0x83236) [0x56032cd38236]
    main
../../../Source/driver/main.cpp:143:5

13: /lib64/libc.so.6(__libc_start_main+0xf3) [0x7f76dd6eb1a3]
    __libc_start_main
??:0

14: ./Castro2d.gnu.DEBUG.MPI.ex(+0x1681e) [0x56032cccb81e]
    _start
??:0
```

**Comment by zingale** [comment]
This works fine with 4th order SDC

**Comment by maxpkatz** [comment]
This was broken by #801.

**Comment by zingale** [comment]
This is fixed in 3acbad789b2aa16f43496d9b485fbf4344887337
